README.md

```diff
@@ -99,7 +99,9 @@ All default options
 var options = {
     configuration: {},
     rulesDirectory: null,
-    emitError: true
+    emitError: true,
+    reportLimit: -1,
```

Comment:
I think the default should be 0, which means "not limited". This should also be documented in the README.